erroranalysis/erroranalysis/_internal/cohort_filter.py

```diff
@@ -223,6 +266,16 @@ def _add_filter_cols(self, df, filters):
             df[CLASSIFICATION_OUTCOME] = \
                 self._compute_classification_outcome_data(
                     self.true_y, pred_y, classes)
+        elif has_regression_error:
+            if PRED_Y in df:
+                pred_y = df[PRED_Y]
+            else:
+                # calculate directly via prediction on model
+                pred_y = self.model.predict(
+                    df.drop(columns=[TRUE_Y, ROW_INDEX]))
+            # calculate regression error and add to df
+            df[REGRESSION_ERROR] = \
```

Comment:
nit: try to avoid \ perhaps this is a bit clearer:
```
df[REGRESSION_ERROR] = self._compute_regression_error_data(
    self.true_y, pred_y)
```

Reply:
Done